test/node/constants.test.ts

```diff
@@ -0,0 +1,180 @@
+      | "\x08" Boolean
+      | "\x09" UTC date time
+      | "\x0A" Null value
+      | "\x0B" Regular expression
+      | "\x0C" DBPointer — Deprecated
+      | "\x0D" JavaScript code
+      | "\x0E" Symbol. — Deprecated
+      | "\x0F" JavaScript code w/ scope — Deprecated
+      | "\x10" 32-bit integer
+      | "\x11" Timestamp
+      | "\x12" 64-bit integer
+      | "\x13" 128-bit decimal floating point
+      | "\xFF" Min key
+      | "\x7F" Max key
+     */
+
+    it('64-bit binary floating point should be 0x01', () => {
+      expect(constants.BSON_DATA_NUMBER).to.equal(0x01);
+    });
+    it('UTF-8 string should be 0x02', () => {
+      expect(constants.BSON_DATA_STRING).to.equal(0x02);
+    });
+    it('Embedded document should be 0x03', () => {
+      expect(constants.BSON_DATA_OBJECT).to.equal(0x03);
+    });
+    it('Array should be 0x04', () => {
+      expect(constants.BSON_DATA_ARRAY).to.equal(0x04);
+    });
+    it('Binary data should be 0x05', () => {
+      expect(constants.BSON_DATA_BINARY).to.equal(0x05);
+    });
+    it('Undefined (value) — Deprecated should be 0x06', () => {
+      expect(constants.BSON_DATA_UNDEFINED).to.equal(0x06);
+    });
+    it('ObjectId should be 0x07', () => {
+      expect(constants.BSON_DATA_OID).to.equal(0x07);
+    });
+    it('Boolean should be 0x08', () => {
+      expect(constants.BSON_DATA_BOOLEAN).to.equal(0x08);
+    });
+    it('UTC date time should be 0x09', () => {
+      expect(constants.BSON_DATA_DATE).to.equal(0x09);
+    });
+    it('Null value should be 0x0A', () => {
+      expect(constants.BSON_DATA_NULL).to.equal(0x0a);
+    });
+    it('Regular expression should be 0x0B', () => {
+      expect(constants.BSON_DATA_REGEXP).to.equal(0x0b);
+    });
+    it('DBPointer — Deprecated should be 0x0C', () => {
+      expect(constants.BSON_DATA_DBPOINTER).to.equal(0x0c);
+    });
+    it('JavaScript code should be 0x0D', () => {
+      expect(constants.BSON_DATA_CODE).to.equal(0x0d);
+    });
+    it('Symbol. — Deprecated should be 0x0E', () => {
+      expect(constants.BSON_DATA_SYMBOL).to.equal(0x0e);
+    });
+    it('JavaScript code w/ scope — Deprecated should be 0x0F', () => {
+      expect(constants.BSON_DATA_CODE_W_SCOPE).to.equal(0x0f);
+    });
+    it('32-bit integer should be 0x10', () => {
+      expect(constants.BSON_DATA_INT).to.equal(0x10);
+    });
+    it('Timestamp should be 0x11', () => {
+      expect(constants.BSON_DATA_TIMESTAMP).to.equal(0x11);
+    });
+    it('64-bit integer should be 0x12', () => {
+      expect(constants.BSON_DATA_LONG).to.equal(0x12);
+    });
+    it('128-bit decimal floating point should be 0x13', () => {
+      expect(constants.BSON_DATA_DECIMAL128).to.equal(0x13);
+    });
+    it('Min key should be 0xFF', () => {
+      expect(constants.BSON_DATA_MIN_KEY).to.equal(0xff);
+    });
+    it('Max key should be 0x7F', () => {
+      expect(constants.BSON_DATA_MAX_KEY).to.equal(0x7f);
+    });
+  });
+
+  describe('BSONType enum', () => {
```

Comment:
Tests above are old, no change except for the import
Tests here are new for the enum